components/library/glib/patches/17-python.patch

```diff
@@ -7,8 +7,8 @@
 -python = import('python').find_installation('python3')
 -# used for '#!/usr/bin/env <name>'
 -python_name = 'python3'
-+python = import('python').find_installation('python3.5')
-+python_name = '/usr/bin/python3.5'
++python = import('python').find_installation('python3.9')
```

Comment:
Why we need to explicitly set this to python3.9?  Why is python3 not sufficient?

Reply:
Without this 3 tests more failing.

Reply:
It would be great to have this documented.  Could you please add a comment to this patch stating what tests are failing without forcing python3.9?  Thanks.

Reply:
BTW, when I tested the previous glib version (2.62.6) I found that some tests are unstable.  They usually pass, but sometimes fails.  Maybe this is the case here too.  IOW, maybe additional failed tests are not caused by `python3` here.

Reply:
I can confirm that without the patch the number of failing tests is always higher than with it. I have tested it several time on two machines.